<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LT-CHN-33875" ignored="8" name="Default test" tests="1" failures="1" timestamp="08 Apr 2021 14:21:29 GMT" time="4.124" errors="0">
  <testcase name="getTheListOfRulesForARevision_status200" time="4.042" classname="restassured.automation.testcases.Restassured_Automation_Rules">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at restassured.automation.listeners.ExtentTestManager.statusLogMessage(ExtentTestManager.java:33)
at restassured.automation.testcases.Restassured_Automation_Rules.getTheListOfRulesForARevision_status200(Restassured_Automation_Rules.java:120)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- getTheListOfRulesForARevision_status200 -->
  <testcase name="deleteARuleSet_status400" time="0.0" classname="restassured.automation.testcases.Restassured_Automation_Rules">
    <ignored/>
  </testcase> <!-- deleteARuleSet_status400 -->
  <testcase name="putUpdateANewRuleSet_status200" time="0.0" classname="restassured.automation.testcases.Restassured_Automation_Rules">
    <ignored/>
  </testcase> <!-- putUpdateANewRuleSet_status200 -->
  <testcase name="putUpdateANewRuleSet_status400" time="0.0" classname="restassured.automation.testcases.Restassured_Automation_Rules">
    <ignored/>
  </testcase> <!-- putUpdateANewRuleSet_status400 -->
  <testcase name="putUpdateANewRuleSet_status409" time="0.0" classname="restassured.automation.testcases.Restassured_Automation_Rules">
    <ignored/>
  </testcase> <!-- putUpdateANewRuleSet_status409 -->
  <testcase name="postCreateANewRuleSet_status409" time="0.0" classname="restassured.automation.testcases.Restassured_Automation_Rules">
    <ignored/>
  </testcase> <!-- postCreateANewRuleSet_status409 -->
  <testcase name="postCreateANewRuleSet_status200" time="0.0" classname="restassured.automation.testcases.Restassured_Automation_Rules">
    <ignored/>
  </testcase> <!-- postCreateANewRuleSet_status200 -->
  <testcase name="postCreateANewRuleSet_status400" time="0.0" classname="restassured.automation.testcases.Restassured_Automation_Rules">
    <ignored/>
  </testcase> <!-- postCreateANewRuleSet_status400 -->
  <testcase name="deleteARuleSet_status204" time="0.0" classname="restassured.automation.testcases.Restassured_Automation_Rules">
    <ignored/>
  </testcase> <!-- deleteARuleSet_status204 -->
</testsuite> <!-- Default test -->
